includes/User.php

```diff
@@ -2514,6 +2575,8 @@ public function getLocalAttribute($attr, $cityId = null, $sep = "-") {
 		if (!isset($cityId)) {
 			$cityId = $wgCityId;
 		}
+
+		return $this->getGlobalAttribute(self::localToGlobalPropertyName($attr, $cityId, $sep));
```

Comment:
So when we introduce the attribute service, there's one less thing to worry about :P